kolibri/plugins/learn/assets/src/vue/learn-page/index.vue

```diff
@@ -11,24 +11,57 @@
     </content-card>
   </card-grid>
 
+  <div class='button-wrapper'>
+    <button @click='toggle()' v-if='expanded'>
+      <i class="material-icons">&#xE5CE;</i> Show Less
+    </button>
+    <button class='pure-button' @click='toggle()' v-else>
+      <i class="material-icons">&#xE5CF;</i> <span>Show More</span>
+    </button>
+  </div>
+
 </template>
 
 
 <script>
 
+  const N_COLLAPSED = 6;
+  const N_EXPANDED = 12;
+
   module.exports = {
     components: {
       'content-card': require('../content-card'),
       'card-grid': require('../card-grid'),
     },
+    data() {
+      return {
+        expanded: false,
+      };
+    },
+    computed: {
+      contents() {
+        const num = this.expanded ? N_EXPANDED : N_COLLAPSED;
```

Comment:
Nice - we could probably factor this out in the future into another computed property to give more 'responsive' numbers of items.